docs/source/examples/kv_cache_connector.md

```diff
@@ -0,0 +1,113 @@
+# KV Cache Connector
```

Comment:
I think this file should be put in the "features" folder and a link should be added in `docs/source/index.rst`. @Shixiaowei02 @schetlur-nv do you agree?

Reply:
Yes, we have a JIRA to track it: TRTLLM-9199